chat-widget/src/components/headerstateful/interfaces/IHeaderStatefulParams.ts

```diff
@@ -22,4 +22,19 @@ export interface IHeaderStatefulParams {
      */
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     endChat: (adapter: any, skipEndChatSDK?: boolean, skipCloseChat?: boolean, postMessageToOtherTab?: boolean) => Promise<void>;
+
+    /**
+     * draggableEventChannel: Channel to send/receive draggable events
+     */
+    draggableEventChannel?: string;
```

Comment:
We need public documentation on how to enable drag feature, and the necessary steps/tutorial on it. Please include within this PR or create a task and prioritize separately

Reply:
I'll create separate PR/task for this